source/common/http/access_log/access_log_impl.cc

```diff
@@ -72,7 +72,7 @@ FilterPtr FilterImpl::fromJson(Json::Object& json, Runtime::Loader& runtime) {
   }
 }
 
-TraceableRequestFilter::TraceableRequestFilter(Runtime::Loader& runtime) : runtime_(runtime) {}
+TraceableRequestFilter::TraceableRequestFilter(Runtime::Loader&) {}
```

Comment:
since this is not used at all, can you fix ctor and remove this param completely?